libs/portals/admin/service-desk/src/screens/Procurers/Procurers.tsx

```diff
@@ -0,0 +1,80 @@
+import { useLoaderData, useNavigate } from 'react-router-dom'
+import { Company } from '../mockProcures'
+import { IntroHeader } from '@island.is/portals/core'
+import { useNationalId } from '../../hooks/useNationalId'
+import {
+  Box,
+  Button,
+  GridContainer,
+  GridRow,
+  Stack,
+  Text,
+} from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import React from 'react'
+import { ServiceDeskPaths } from '../../lib/paths'
+
+const Procurers = () => {
+  const { formatMessage } = useLocale()
+  const navigate = useNavigate()
+  const { formatNationalId } = useNationalId()
+  const company = useLoaderData() as Company
+
+  return (
+    <Stack space="containerGutter">
+      <Button
```

Comment:
It would be great to create a back button component under `libs/portals/admin/core/src/components/BackButton`

And reuse here and in ids admin